pkg/network/protocols/http/incomplete_stats.go

```diff
@@ -101,20 +115,21 @@ func (b *incompleteBuffer) Add(tx Transaction) {
 	tx = ebpfTxCopy
 
 	if tx.StatusCode() == 0 {
+		b.telemetry.joiner.requests.Add(1)
 		parts.requests = append(parts.requests, tx)
 	} else {
+		b.telemetry.joiner.responses.Add(1)
 		parts.responses = append(parts.responses, tx)
 	}
 }
 
-func (b *incompleteBuffer) Flush(now time.Time) []Transaction {
+func (b *incompleteBuffer) Flush(nowUnixNano int64) []Transaction {
```

Comment:
the pr description does not explain this change (there is a section about timestamp, but does not reflect the actual change)

Reply:
I will update the description